config.defaults.yaml

```diff
@@ -3,6 +3,7 @@ kowalski:
   port: 443
   protocol: https
   token:
+  alt_token:
```

Comment:
Maybe document what this corresponds to?

Reply:
Good idea, thanks!